source/object-model/computed-properties.md

```diff
@@ -70,10 +72,12 @@ You surround the dependent properties with braces (`{}`), and separate with comm
 This is especially useful when you depend on properties of an object, since it allows you to replace:
 
 ```javascript
-let obj = Ember.Object.extend({
+import EmberObject, { computed } from '@ember/object';
+
+let obj = EmberObject.extend({
   baz: {foo: 'BLAMMO', bar: 'BLAZORZ'},
 
-  something: Ember.computed('baz.foo', 'baz.bar', function() {
+  something: Embercomputed('baz.foo', 'baz.bar', function() {
```

Comment:
One of the `Embercomputed` conversions ^ got missed.

Reply:
Thanks :)